README.md

```diff
@@ -257,18 +257,18 @@ Error messages can also be constructed on the fly to customize how the schema fa
 
 ```ts
 const order = object({
-  no: number().required().
+  no: number().required(),
   sku: string().test({
     name: 'is-sku',
     skipAbsent: true,
     test(value, ctx) {
-      if (!value.startsWith('s-')) {
+      if (!value!.startsWith('s-')) {
         return ctx.createError({ message: 'SKU missing correct prefix' })
       }
-      if (!value.endsWith('-42a')) {
+      if (!value!.endsWith('-42a')) {
         return ctx.createError({ message: 'SKU missing correct suffix' })
       }
-      if (value.length < 10) {
+      if (value!.length < 10) {
```

Comment:
I'd rather not include the TS `!` here as it makes it harder to copy for folks using plain JS